initial-configuration/jenkins/jenkins-docker/jobs/PIC-SURE Pipeline/config.xml

```diff
@@ -22,9 +22,9 @@ def build_hashes = {
     UPLOADER: false
 };
 pipeline {
-    agent any
```

Comment:
There are no non whitespace changes in this file. Please revert.

Reply:
Done.